test/e2e/integration/cloud_backup_test.go

```diff
@@ -127,6 +129,29 @@ var _ = Describe("CloudBackup", func() {
 			Container: os.Getenv("TEST_ID"),
 			Prefix:    path.Join("v2"),
 		}
+		// Create and place a ETCD config yaml
+		outfile := "/tmp/etcd.conf.yaml"
+		etcdConfigYaml := `# Human-readable name for this member.
+    name: etcd1
+    data-dir: ` + os.Getenv("ETCD_DATA_DIR") + `
+    metrics: extensive
+    snapshot-count: 75000
+    enable-v2: false
+    quota-backend-bytes: 8589934592
```

Comment:
```suggestion
    quota-backend-bytes: 1073741824
```
as it is just for testing we can decrease the quota bytes?

Reply:
Done